frontend/packages/app/src/clutch.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
       trending: true,
       componentProps: {
         resolverType: "clutch.aws.ec2.v1.Instance",
+        note: "The instance may take several minutes to shut down.",
```

Comment:
why not array of notes into note panel? for consistency

the confirmation design in figma is missing the incorporation of severity which we will end up needing

Reply:
This was to match the designs in figma. I figured all the config notes would move to this interface in the future. I wasn't sure if going forward we would have these be a list or a single string to match figma

Reply:
per discussion earlier, let's go back to a list w sev since the designer will be updating the designs to incorporate the possibility of multiple notes.